test/find.test.js

```diff
@@ -12,30 +12,30 @@ describe('SourceMap - Find', () => {
 
     let mapping = map.findClosestMapping(2, 8);
     assert.deepEqual(mapping, {
-      generated: { line: 2, column: 9 },
-      original: { line: 1, column: 7 },
-      source: './helloworld.coffee',
+      generated: { line: 2, column: 2 },
+      original: { line: 1, column: 0 },
+      source: 'helloworld.coffee',
```

Comment:
This change was intentional?

Reply:
Yes, wanted to make it follow the same logic as `source-map` library, esbuild, ...

Reply:
I can revert it very easily